.github/workflows/r-python-interop-testing.yml

```diff
@@ -107,12 +111,13 @@ jobs:
       - name: Update Packages
         run: Rscript -e 'update.packages(ask=FALSE)'
```

Comment:
We should remove these two lines as we did in the other yaml file.  I can send a follow-up commit.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/2374/commits/3cb2e01a228d626adf1f827450940435a4b7dbec